docs/welcome.md

```diff
@@ -0,0 +1,30 @@
+---
+id: welcome
+title: Welcome to Hubs
+sidebar_label: Welcome
+---
+
+## What is Hubs?
+Hubs is a social VR platform that runs right in your browser. With Hubs you can create your own virtual space with a single click and invite others to join using a URL. No installation process. No app store. No gatekeepers.
+
+Watch videos with friends, give presentations, or meet with colleagues remotely. Hubs makes it easy to connect with others and share images, videos, 3D models, and more. 
+
+Hubs works across platforms. Got a VR headset? Awesome! If not, you can use your computer or mobile device.
```

Comment:
I wonder if we should enumerate what we mean by the various platforms. e.g. "... use your desktop computer, laptop, tablet, or mobile device". Not sure if this is an improvement. Feel free to disregard because I suspect I am in the minority in thinking that it's odd to call desktop/laptops "computers" in contrast to a mobile device (which, to me, is also a "computer").

Reply:
I agree with this! I'll update this text.